obspy/core/tests/test_waveform_plugins.py

```diff
@@ -61,7 +61,7 @@ def test_read_and_write(self):
         for format in formats:
             # XXX: skip SEGY and SU formats for now as they need some special
             # headers.
-            if format in ['SEGY', 'SU', 'SEG2']:
+            if format in ['SEGY', 'SU', 'SEG2', 'AH']:
```

Comment:
please don't skip here, but only below where the `calib` is checked, line number as indicated by the link I posted.